source/css/_variables/base.styl

```diff
@@ -134,8 +131,6 @@ $table-row-hover-bg-color       = $whitesmoke
 // Code & Code Blocks
 // --------------------------------------------------
 $code-font-family               = $font-family-monospace
-$code-font-size                 = 14px
-$code-font-size                 = unit(hexo-config('font.codes.size'), px) if hexo-config('font.codes.size') is a 'unit'
```

Comment:
Why do u remove this?

Reply:
Reason: https://github.com/theme-next/hexo-theme-next/issues/687
`$code-font-size` should not exist, because `font-size` of `<code>` must be `1em`